vernac/comFixpoint.mli

```diff
@@ -19,12 +19,10 @@ open Vernacexpr
 (** Entry points for the vernacular commands Fixpoint and CoFixpoint *)
 
 val do_fixpoint :
-  (* When [false], assume guarded. *)
-  locality -> polymorphic -> (fixpoint_expr * decl_notation list) list -> unit
+  check_guard -> locality -> polymorphic -> (fixpoint_expr * decl_notation list) list -> unit
```

Comment:
The removal of the comment is only making things less clear because the semantics of `check_guard` is non-obvious.